### DISCUSSION TITLE
Inline editing of table row v7

### DISCUSSION THREAD

**Discussion by Nedu** [TanStack/table#3258] [Q&A]
I am trying to create a table that can be an editable per row when clicking on an edit icon button in a table cell. I also want to make it possible that when a row is being edited the edit icon button changes to a save icon button and a cancel icon button. Here is my code:
https://codesandbox.io/s/interesting-resonance-nfyjs?file=/src/App.tsx

**Comment by xjackk** [comment]
There is an example that will get you almost all the way there right on the v7 docs.
https://react-table-v7.tanstack.com/docs/examples/editable-data
